### PR TITLE
Document how to change environment variables

### PR DESCRIPTION
**Pull request by tahb**
- Document how to view, change and apply environment variables on the live GPaaS environments.
- We make a minor addition to local environment management to include reference to our Dotenv initialiser for use with critical variables whose absense would prevent the app starting.

### CODE REVIEW

File: doc/managing-environment-variables.md [hunk 1]
```diff
@@ -8,3 +10,5 @@ To manage sensitive environment variables:
 
 1. Add the new key and safe default value to the `/.env.example` file eg. `ROLLBAR_TOKEN=ROLLBAR_TOKEN`
 2. Add the new key and real value to your local `/.env.development.local` file, which should never be checked into Git. This file will look something like `ROLLBAR_TOKEN=123456789`
+
+If the environment is critical whereby it is required to start the application, please add it to [the Dotenv initialiser](../config/initializers/_dotenv.rb).
```
Comment (lozette) [inline doc/managing-environment-variables.md:14]: Missing word `If the environment VARIABLE...`

**Review by lozette — APPROVED** [review]
Approved but one small change required

File: doc/managing-environment-variables.md [hunk 2]
```diff
@@ -8,3 +10,54 @@ To manage sensitive environment variables:
...
+
+#### Application
+
+A variable will usually be required to be added into each environment through [a new repository secret](https://github.com/DFE-Digital/buy-for-your-school/settings/secrets/actions/new). For this we use prefixes in our environment variable names:
+
+```
+APP_ENV_PROD_
+APP_ENV_STAGING_
+APP_ENV_RESEARCH_
+APP_ENV_PREVIVEW_
```
Comment (jacksonj04) [inline doc/managing-environment-variables.md:32]:
```suggestion
APP_ENV_PREVIEW_
```

**Review by jacksonj04 — CHANGES_REQUESTED** [review]
One tiny tweak to make sure the docs are correct!

**Comment by tahb** [comment]
2 extra commits that I fixed as CI was failing intermittently (after turning tests back on in CI presumably). Piggy backing on this PR.

**Review by jacksonj04 — APPROVED** [review]

**Pull request merged**